### PR TITLE
feat: add doc on notification in yaml files

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: docs/content/docs/concepts/files/workflow-syntax.md [hunk 1]
```diff
@@ -71,3 +78,51 @@ Example with using LUA syntax as advanced condition:
...
+```
+
+Example of jabber notification. Note that you can add `conditions` on every notifications
+
+```yml
+- type: jabber
+  pipelines:
+  - deploy
+  settings:
+    on_start: true`
```
Comment (yesnault) [inline docs/content/docs/concepts/files/workflow-syntax.md:105]:
```suggestion
    on_start: true
```

File: docs/content/docs/concepts/files/workflow-syntax.md [hunk 2]
```diff
@@ -71,3 +78,51 @@ Example with using LUA syntax as advanced condition:
...
+    send_to_groups: true
+    recipients:
+    - me@jabber.com
+    conditions:
+      check:
+      - variable: cds.triggered_by.email
+        operator: eq
+        value: me@ovh.com
```
Comment (yesnault) [inline docs/content/docs/concepts/files/workflow-syntax.md:113]:
```suggestion
        value: me@localhost.local
```

File: docs/content/docs/concepts/files/workflow-syntax.md [hunk 3]
```diff
@@ -71,3 +78,51 @@ Example with using LUA syntax as advanced condition:
...
+```
+
+Example of vcs notification. Note that `pipelines` list is optionnal on every notifications. When it's not specified, notification will be triggered for each pipeline
```
Comment (yesnault) [inline docs/content/docs/concepts/files/workflow-syntax.md:116]:
```suggestion
Example of vcs notification. Note that `pipelines` list is optional on every notifications. When it's not specified, notification will be triggered for each pipeline
```

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12926.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✘  
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Pull request merged**